comp/dogstatsd/bundle.go

```diff
@@ -19,4 +20,5 @@ var Bundle = fxutil.Bundle(
 	serverdebugimpl.Module,
 	replay.Module,
 	server.Module,
+	statsd.Module,
```

Comment:
I personally think this should be moved to `core` because it makes more sense next to `core/log` and `core/telemetry` since this will be used mainly *outside* of the agent-metrics-logs and mostly by other agents.

I'm happy to move it back